protocol/kafka_confluent/v2/option.go

```diff
@@ -78,6 +78,14 @@ func WithSender(producer *kafka.Producer) Option {
 	}
 }
 
+// WithEventHandler provide a func on how to handle the kafka.Event for the producer.Events() chan. This option is not required.
```

Comment:
nit: can you please remove all the "This option is not required" suffixes from the options? Missed that in the other review. Options are never required.

Reply:
Done.